Source/DafnyLanguageServer/Handlers/DafnyHoverHandler.cs

```diff
@@ -215,15 +215,30 @@ class AssertionBatchIndexComparer : IComparer<AssertionBatchIndex> {
 
       string GetDescription(Boogie.ProofObligationDescription? description) {
         switch (assertionNode?.StatusVerification) {
-          case GutterVerificationStatus.Verified:
-            return $"{obsolescence}<span style='color:green'>**Success:**</span> " +
-                   (description?.SuccessDescription ?? "_no message_");
+          case GutterVerificationStatus.Verified: {
+              var successDescription = description?.SuccessDescription ?? "_no message_";
+              if (successDescription == "error is impossible: This is the precondition that might not hold.") {
```

Comment:
Could you refer to the substrings from which this concatenated string originates?

Reply:
> Could you refer to the substrings from which this concatenated string originates?

I actually found a way to get rid of this if statement by adding one more proof obligation description that ought to be implemented.